public/resources/core/js/controllers/product-offering.controller.js

```diff
@@ -385,6 +447,26 @@
             });
         }
 
+        function togglePriceAlteration() {
+            switch (vm.priceAlteration) {
+            case 2:
+                vm.pricePlan.productOfferPriceAlteration = new Offering.PriceAlteration(vm.pricePlan, {
+                    name: vm.PRICE_ALTERATIONS.FEE.code
+                });
+                break;
+            case 1:
+                vm.pricePlan.productOfferPriceAlteration = new Offering.PriceAlteration(vm.pricePlan, {
+                    name: vm.PRICE_ALTERATIONS.FEE.code,
+                    priceCondition: 0,
+                    percentageOrPrice: false,
+                    priceConditionOperator: vm.PRICE_CONDITIONS.GE.code
+                });
+                break;
+            default:
+                vm.pricePlan.productOfferPriceAlteration = null;
+            }
+        }
+
```

Comment:
It seems that the togglePriceAlteration method is the same in all the controllers. is it not possible to have an auxiliary method?